src/ml_tooling/metrics.py

```diff
@@ -70,6 +71,44 @@ def confusion_matrix(y_true: np.ndarray, y_pred: np.ndarray, normalized=True) ->
     return cm
 
 
+def permuted_feature_importance(model,
+                                x,
+                                y,
+                                refit=True,
+                                metric=None):
+    """
+    Calculates feature importance by randomly permuting features and comparing result to baseline
+    :param y:
```

Comment:
Isn't y training/test target?

Reply:
Yeah, changed the implementation without updating docs - still have some work to do on this feature 😄